packages/site/src/layout/SearchBox.tsx

```diff
@@ -45,7 +49,7 @@ export const SearchBox = ({
 
   const filterFunction = (item: ContentListItem) =>
     item.title.toLowerCase().includes(search.toLowerCase()) ||
-    item.additionalMatches?.find((e: string) =>
+    item.additionalMatches?.some((e: string) =>
```

Comment:
I feel like `.some` is all we need here. `.some` is just looking for if any `additionalMatches` match the search term and that's all we care about.
`.find` is going to return the matching element which we don't need in this case.